docs/reference/query-dsl/parent-id-query.asciidoc

```diff
@@ -1,68 +1,113 @@
 [[query-dsl-parent-id-query]]
 === Parent Id Query
 
-The `parent_id` query can be used to find child documents which belong to a particular parent.
-Given the following mapping definition:
+Returns child documents <<parent-join,joined>> to a specific parent document.
+You can use a <<parent-join,join>> field mapping to create parent-child
+relationships between documents in the same index.
 
+[[parent-id-query-ex-request]]
+==== Example request
+
+[[parent-id-index-setup]]
+===== Index setup
+To use the `parent_id` query, your index must include a <<parent-join,join>>
+field mapping. To see how you can set up an index for the `parent_id` query, try
+the following example.
+
+. Create an index with a <<parent-join,join>> field mapping.
```

Comment:
`.` dot at the beginning of the sentence?
Here and in a couple of places below.

Reply:
This renders as an ordered list in the HTML output. Here's a screenshot:

<img width="735" alt="Screen Shot 2019-07-19 at 10 35 31 AM" src="https://user-images.githubusercontent.com/40268737/61543211-05347280-aa11-11e9-97d4-546d42efe9df.png">